### PR TITLE
Prepare minor version 4.47

### PR DESCRIPTION
**Pull request by northwestwitch**
### Added
- [x] Added CADD, GnomAD and genotype calls to variantS export
### Changed
- [x] Pull request template, to illustrate how to deploy pull request branches on cg-vm1 stage server
### fixed
- [x] Compiled Docker image contains a patched version (v4.9) of chanjo-report

**Review:**
- [x] code approved by DN
- [x] tests executed by DN. CR

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
-## []
+## [4.47]
 ### Added
 - Added CADD, GnomAD and genotype calls to variantS export
 ### Changed
 - Pull request template, to illustrate how to deploy pull request branches on cg-vm1 stage server
+### fixed
```
Comment (dnil) [inline CHANGELOG.md:12]:
```suggestion
### Fixed
```

Comment (dnil) [reply CHANGELOG.md]: 🧐

**Comment by northwestwitch** [comment]
I'll install on cg-vm1 to check that the chanjo-report bug is fixed!

**Review by dnil — APPROVED** [review]

**Comment by northwestwitch** [comment]
Yepp: 🆗 
image

**Comment by dnil** [comment]
🆗 VariantS export update as well:
Screenshot 2022-01-21 at 11 04 19

**Comment by dnil** [comment]
Though the default precision on those cells.. Oh well.

**Comment by northwestwitch** [comment]
> Though the default precision on those cells.. Oh well.

Looks very precise to me! 😆

**Comment by dnil** [comment]
:ok:
Deployed to Solna instance:
```
[hiseq.clinical@cg-prod-services servers]$ systemctl --user start scout@v4.47
```

**Pull request merged**